### PR TITLE
[HTML5] - Fix for the React console warnings

### PR DESCRIPTION
**Pull request by OZhurbenko**
Fixed console warnings that appear for viewers and / or moderators on initial login, when they open a user list and chat.

All except one. There is a custom prop validator in the DropdownList, which will need to be adjusted later.

### CODE REVIEW

**Comment by MaximKhlobystov** [comment]
Great work! You made my day. Tested this, looks good. There's only one warning left: `"Warning: Failed prop type: Invalid prop `children[2]` supplied to `DropdownList`. Validation failed."` The rest is gone.

File: bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx [hunk 1]
```diff
@@ -41,7 +41,17 @@ export default class DropdownTrigger extends Component {
   }
 
   render() {
-    const { children, className, ...restProps } = this.props;
+    const remainingProps = Object.assign({}, this.props);
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx]: `const remainingProps = { ...this.props }`

Comment (OZhurbenko) [reply bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx]: Hey, I was using [your own method](https://github.com/bigbluebutton/bigbluebutton/blame/v2.0.x-release/bigbluebutton-html5/imports/ui/components/button/component.jsx#L129) to remove the unknown props here.

Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx]: I was not complaining about the deletion of props.

**Pull request merged**